<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="btn_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>391</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_50">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_100">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_500">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_coffee">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee(100)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tea">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tea(150)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_milk">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Milk(200)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
